ChangeLog

```diff
@@ -2,6 +2,10 @@ mbed TLS ChangeLog (Sorted per branch, date)
 
 = mbed TLS x.x.x branch released xxxx-xx-xx
 
+Bugfix
+   * Fix memory leak and free without initialization in pk_encrypt
```

Comment:
Do we add to ChangeLog fixes in the sample applications?

Reply:
@RonEld I don't know, honestly. @andresag01 ?

Reply:
Maybe because it was reported by the community, it would be nice, for the credit

Reply:
Yes, for the credit, we should keep it in here, but in general, if it's a fix we find and introduce, I don't know.

Reply:
I do not really know about that. I do not think we have added ChangeLog entries for this in the past.

Reply:
If we found and fixed this ourselves it wouldn't deserve ChangeLog entry, but as Hanno says, since it was reported externally we want one so we can give credit. The bar for ChangeLog entries is lower for external contributions. (If someone just fixes a one-character typo in a comment, this probably doesn't deserve an entry, but here I think it does.)